web_full_stack/python/app.py

```diff
@@ -0,0 +1,22 @@
+from flask import Flask, request, render_template
+from reverb_client import ReverbClient
+app = Flask(__name__)
+
+@app.route('/')
+@app.route('/categories')
+def categories():
+  categories = __search_categories(request.args.get('query'))
+  return render_template('categories.html', categories=categories)
+
+@app.route('/listings')
+def listings():
+  return render_template('listings.html', listings=ReverbClient().listings())
+
+def __search_categories(query):
```

Comment:
minor pythonic thing - single underscore for private methods - https://docs.python-guide.org/writing/style/#we-are-all-responsible-users